Apps/AdminWebClient/src/Server/Controllers/CommunicationController.cs

```diff
@@ -84,5 +81,19 @@ public IActionResult GetAll()
         {
             return new JsonResult(this.communicationService.GetAll());
         }
+
+        /// <summary>
+        /// Deletes the Communication that is passed to the controller.
+        /// </summary>
+        /// <param name="communication">The communication object to delete.</param>
+        /// <returns></returns>
+        /// <response code="200">Returns the communication json of the deleted object wrapped in a request result.</response>
+        /// <response code="401">the client must authenticate itself to get the requested response.</response>
+        /// <response code="403">The client does not have access rights to the content; that is, it is unauthorized, so the server is refusing to give the requested 
+        public IActionResult Delete([FromBody] Communication communication)
```

Comment:
As this CommunicationController has the [ApiController] attribute, I don't think we need to specify the "[FromBody]" attribute for this complex type param. But it is no harm.